### ISSUE TITLE
how can i integrate my GPS device?

### ISSUE THREAD

**Issue by jacksolutions** [evigra/gpsmaps#1]
i have installed this module, created geofences as well. i have also added IMEI and phone number in vehicle, but how my mobile will send GPS data to odoo? is there any configuration to do? i am confused? please assist me and thank you for the great module.

**Comment by jacksolutions** [comment]
also i got the error "geocoder failed due to over query limit", i am not sure why?

**Comment by jacksolutions** [comment]
can you please assist us? we are stuck with this module.

Thanks

**Comment by evigra** [comment]
Hello Friend.
The module is only frontend.
You need a backend
Your GPS should see the backend
The backend will add data to Odoo

**Comment by jacksolutions** [comment]
I have seen that you have used traccar for GPS back end ?

Would you please help me or any setup guide for the back end to connect ?

**Comment by evigra** [comment]
"geocoder query failed due to over limit"
This message is sent by google.
and it is for that 1 IP performed over 25,000 requests in the day.

**Comment by evigra** [comment]
That's right.
use Traccar
Friend, I have one year to relate GPS, GMaps, Odoo, configure routers.
You should paid 3000 dollars for configuration(Only Traccar and GPSMAPS).

**Comment by jacksolutions** [comment]
"geocoder query failed due to over limit"
This message is sent by google.
and it is for that 1 IP performed over 25,000 requests in the day.

=> i dont think so that this addon has requested 25k request in a day cause
after installation i have not used that system cause i was struggling
linking it with backend.

On Sun, Aug 23, 2015 at 9:27 AM, Eduardo Vizcaino <notifications@github.com>
wrote:

> "geocoder query failed due to over limit"
> This message is sent by google.
> and it is for that 1 IP performed over 25,000 requests in the day.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/evigra/gpsmaps/issues/1#issuecomment-133778725>.
>



-- 
Cheers,

Jaykishan Vansadawala
(Jack Solutions)
Mob:- +91-9879455635

**Comment by jacksolutions** [comment]
are you saying that you need 3000 dollars for configuration? unfortunately
i am not that reach.. :( but i will try to do configuration my way. thank
you for your help friend and i appreciate it.

thank you for the great plugin. if i will get any client who wants to do
some customization related to GPSMAPS then definitely i will route them to
you.



On Sun, Aug 23, 2015 at 9:54 AM, Eduardo Vizcaino <notifications@github.com>
wrote:

> That's right.
> use Traccar
> Friend, I have one year to relate GPS, GMaps, Odoo, configure routers.
> You should paid 3000 dollars for configuration(Only Traccar and GPSMAPS).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/evigra/gpsmaps/issues/1#issuecomment-133779892>.
>



-- 
Cheers,

Jaykishan Vansadawala
(Jack Solutions)
Mob:- +91-9879455635

**Comment by evigra** [comment]
You have to configure the backend to Odoo.
You have to configure your router to Odoo
and you have to configure the GPS to Odoo

You have installed the client on your phone?

**Comment by jacksolutions** [comment]
Yes I have installed traccar client on android phone and traccar server on
my vps.

**Comment by evigra** [comment]
Do you have a VPS?
You do not have your own server?

You are behind a router?
If so, you must configure ports. and your provider will not do it

**Comment by jacksolutions** [comment]
I have server on digital ocean cloud.. So i don't think I have to do any
router configuration ?

What other settings I have to do to make it work ?

**Comment by evigra** [comment]
You need to configure the GPS to see Traccar.
You need to configure the Traccar, to see Odoo

Between GPS and tracar, you must configure ports.

**Comment by tymiles003** [comment]
Sir,
How do I setup traccar to see odoo?

**Comment by evigra** [comment]
1. Installs GPSMAPS
2. traccar is configured with the needs of GPSMAPS

**Comment by tymiles003** [comment]
How do I install traccar to work with odoo, is there an installation
process I can follow?

**Comment by buguelos** [comment]
hi dear fantastic module i got on the same server traccar and odoo i setup traccar on the same database of odoo but i coulnd find any way to traccar see odoo i didn't understand how to on traccar web interface i can see my devices but on odoo gpsmaps module still not seeing it can you help on traccar xml configuration???

**Comment by evigra** [comment]
I'm sorry, I speak very little English.
You can see localhost:8082            ? traccarweb

**Comment by Salman-Jawad91** [comment]
Thank you for the useful module.

I have the same issue that I am able to configure traccar to get the gps data from my mobile., but how to connect  odoo wih traccar , still not working. I have adjusted the traccar.xml file to be like:

<entry key='database.driver'>org.postgresql.Driver</entry>
<entry key='database.url'>jdbc:postgresql://IP-ADDREESS:5432/DATABASENAME</entry>
<entry key='database.user'>USERNAME</entry>
<entry key='database.password'>PASSWORD</entry>

I have open all the connection to the postgresql database , but still this not solved.
Any help !

**Comment by evigra** [comment]
Goodnight.
I have a vehicle tracking company ...
we could generate the communication between your server and our company

**Comment by Salman-Jawad91** [comment]
Hello Evigra


Thanks for your prompt reply and thanks again for sharing the source code.


I have already setup a traccar on my server and install a client on a cell phone and it is working fine , and I setup odoo as well and install GPSMAPs module  but the thing is I am unable to bring the gps data from traccar to odoo. I changed the traccar.xml but it is not working. do I need to change the system paramaters in odoo ?? and what will be the value  ??


Your help is highly appreciated.


Thanks

________________________________

**Comment by evigra** [comment]
Goodnight.
Traccar, are you working?
If Traccar, is working ... What's the problem?

**Comment by Salman-Jawad91** [comment]
Hello evigra,


Traccer server is working ,but the connection between odoo and traccar is not working I have added the lmei and phone number and still it is not bringing the data from traccar to odoo. Can you please help in what other configuration needs to be done on odoo


Thanks !


________________________________

**Comment by adiska82** [comment]
my traccar is good, i see this step, does anyone know, where i can find this step ?


![fleet_tracking_settings-odoo](https://user-images.githubusercontent.com/58172133/75530533-606a2100-5a46-11ea-8005-7de52baa89c1.png)
